src/si5351.h

```diff
@@ -33,8 +33,23 @@
 #include "Wire.h"
 #include <stdint.h>
 
+//#define SI5351_WITH_CLK67 0
+
+#ifndef SI5351_WITH_CLK67
+#define SI5351_WITH_CLK67 1
+#endif
+#ifndef SI5351_WITH_NEW
+#define SI5351_WITH_NEW 0
+#endif
+
 /* Define definitions */
 
+#if WITH_CLK67
+#define SI5351_CLK_LAST SI5351_CLK7
+#else
+#define SI5351_CLK_LAST SI5351_CLK5
+#endif
+
```

Comment:
Do we really need to set 2 seperate flags(`SI5351_WITH_CLK67`, `WITH_CLK67`) and  to indicate that there are clocks 6 and 7? It can be a single flag I think.

Reply:
I think `WITH_CLK67` should change to `SI5351_WITH_CLK67`.